tutorials/marble/explore-the-network.md

```diff
@@ -0,0 +1,46 @@
+# Explore the Network
+
+```{contents}
+:local:
+```
```

Comment:
I suggest removing this local contents section from all pages for two reasons: 1. in Jupyter book, local TOC is anyways shown by default on the right hand side, 2. Adding this TOC makes the section titles look like hyperlinks which are empty, and it doesn't look nice to have hyperlinks in huge font size. See the two before and after figures below.

<img width="1459" alt="Screenshot 2024-01-02 at 12 06 40 AM" src="https://github.com/DACCS-Climate/marble-tutorials/assets/5779123/910c4970-4d26-4c8f-9292-7e9bc2d23756">
<img width="1492" alt="Screenshot 2024-01-02 at 12 07 07 AM" src="https://github.com/DACCS-Climate/marble-tutorials/assets/5779123/86fea274-2f46-48be-a9c9-6d958ff45a5d">

Reply:
Removed the local contents from explore-the-network.md, account-management.md, and get-an-account.md.